api_spec.yaml

```diff
@@ -809,30 +810,22 @@ paths:
         content:
           application/json:
             schema:
-              oneOf:
-                - type: object
-                  properties:
-                    redirectUris:
-                      type: array
-                      items:
-                        $ref: '#/components/schemas/oauth2RedirectUri'
-                    name:
-                      type: string
-                      example: asdf
-                    scopes:
-                      type: array
-                      items:
-                        $ref: '#/components/schemas/oauth2Scope'
-                  required:
-                    - redirectUris
-                    - name
-                - type: object
-                  properties:
-                    status:
-                      type: string
-                      enum: ["UNKNOWN_SCOPE"]
-                    scope:
-                      $ref: '#/components/schemas/oauth2Scope'
+              type: object
+              properties:
+                redirectUris:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/oauth2RedirectUri'
+                name:
+                  type: string
+                  example: 'developement'
```

Comment:
name in request example not matching response..

Reply:
changes made